week2/jalalhm/test2.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+
+const fs = require('fs');
+
+const TODOS_FILENAME = process.env["TODOS_FILENAME"] || "todos.json";
```

Comment:
Nice use of an env var with a default value (unless the filename is "0" or another falsey value).